modules/local/mt_deletion_scritp.nf

```diff
@@ -0,0 +1,43 @@
+process MT_DELETION {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::samtools=1.16.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.16.1--h6899075_1' :
+        'quay.io/biocontainers/samtools:1.16.1--h6899075_1' }"
+
+    input:
+    tuple val(meta), path(stats)
+
+    output:
+    path '*.txt'       , emit: mt_del_script
+    path "versions.yml", emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    grep -E ^IS $stats | awk 'BEGIN {sum=0} (\$2>=1200 && \$2<=15000) {sum=sum+\$3} (\$2<1200 || \$2>15000) {sum_norm=sum_norm+\$3} END \\
+    {print "intermediate discordant ", sum, "normal ", sum_norm, "ratio ppk", sum*1000/(sum_norm+sum)}' 1> ${prefix}_mt_del.txt
```

Comment:
```suggestion
    {print "intermediate discordant ", sum, "normal ", sum_norm, "ratio ppk", sum*1000/(sum_norm+sum)}' 1> ${prefix}.txt
```
Let's try not to hardcode the endings. We can add the ending to the config or if we we need it to be in the module you could potentially set it on line 21 like this. `def prefix = task.ext.prefix ?: "${meta.id}_mt_del.txt"`. This way the user has full control on what to call the file. Do you have an opinion on this @ramprasadn?

Reply:
Yupe, I agree.